addons/dark-www/addon.json

```diff
@@ -2097,9 +2097,9 @@
   "injectAsStyleElt": true,
   "versionAdded": "1.4.0",
   "latestUpdate": {
-    "version": "1.23.0",
-    "isMajor": true,
-    "temporaryNotice": "You can now completely customize the appearance of the Scratch website."
+    "version": "1.33.2",
+    "temporaryNotice": "TODO.",
```

Comment:
```suggestion
    "temporaryNotice": "The presets for this addon have been updated to use Scratch's new purple accent color. Presets that use blue are still available as well.",
```

Reply:
**THIS TEMPORARY NOTICE WOULD ALSO BE ADDED TO THE editor-dark-mode ADDON.**